bsp/chips/cc1200/radio.c

```diff
@@ -138,18 +138,28 @@ void radio_rfOff(void) {
 
 //===== TX
 
-void radio_loadPacket(uint8_t* packet, uint8_t len) {
+void radio_loadPacket(uint8_t* packet, uint16_t len) {
+    //test 802.15.4g PHR. This has to be done by the MAC layer
+    uint8_t  PHR[2];  
+ 
+    if (len<2048){
```

Comment:
If's cleaner to do error cheking at the beginning of the function, and return if not OK. Then your normal code is not in an IF statement.